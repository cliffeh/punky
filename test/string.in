(substr "abcdefg" 2 3)           ; "cde"
(strlen "abcdefg")               ; 7
(split "this is the string" " ") ; ("this" "is" "the" "string")
