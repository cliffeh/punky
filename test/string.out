"cde"
7
("this" "is" "the" "string")
